app/code/community/SomethingDigital/Modal/Block/Modal.php

```diff
@@ -12,6 +12,9 @@ public function getConfig($value)
     public function getModalContent() 
     {
         $blockId = $this->getConfig('block');
+        if(!$this->getLayout()) {
+            return;
+        }
```

Comment:
Do we have this change deployed to staging? https://github.com/sdinteractive/SomethingDigital_Modal/pull/9/files